CONTRIBUTE.md

```diff
@@ -18,33 +18,32 @@ This collection requires **Python v3.6 or greater**. If you don't have Python in
 
 
 Current Ansible-core and Ansible version supported:
-- `ansible-core`: **2.11**
-- `ansible`: **4.0**
+- `ansible-core`: **2.11.x**
+- `ansible`: **4.0.x**
 
-You may install Ansible in a `virtualenv` or globally. 
+We recommend installing Ansible in a `virtualenv` created specifically for this project. 
 
 #### Install Ansible via virtualenv
 
 ```bash
 python3 -m venv <path_to_venv>/onepassword_ansible
 source <path_to_venv>/onepassword_ansible activate
 
-pip3 install ansible-core==2.11.*
-pip3 install ansible==4.0.*
-```
-
-#### Install Ansible System-wide 
-```bash
-pip3 install ansible-core==2.11.*
-pip3 install ansible==4.0.*
+pip3 install ansible-core==2.11.* ansible==4.0.*
```

Comment:
Maybe we should stick with specific versions, in the bash command.
Running this command as is triggers an error:
```
% pip3 install ansible-core==2.11.* ansible==4.0.*

zsh: no matches found: ansible-core==2.11.*
```

Reply:
Good point, I must've forgotten the syntax for Python requirements 😄 

I'll change it to use the correct [requirement specifiers](https://pip.pypa.io/en/stable/cli/pip_install/#requirement-specifiers)